### PR TITLE
add: `minecraft-pi-reborn-app`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
This should be a `-app`, not `-bin`

File: packages/minecraft-pi-reborn-app/minecraft-pi-reborn-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name="minecraft-pi-reborn-bin"
+pkgname="minecraft-pi-reborn"
+version="2.3.5"
+gives="${pkgname}"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-deb ${pkgname}-app minecraft-pi-reborn-client"
```
Comment (Henryws) [inline packages/minecraft-pi-reborn-app/minecraft-pi-reborn-bin.pacscript:5]: Can't break itself

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/minecraft-pi-reborn-app/minecraft-pi-reborn-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name="minecraft-pi-reborn-bin"
+pkgname="minecraft-pi-reborn"
```
Comment (Henryws) [inline packages/minecraft-pi-reborn-app/minecraft-pi-reborn-bin.pacscript:2]:
```suggestion
name="minecraft-pi-reborn-app"
pkgname="minecraft-pi-reborn"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — CHANGES_REQUESTED** [review]
Rename the pacscript to be `-app`

File: packages/minecraft-pi-reborn-app/minecraft-pi-reborn-app.pacscript [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+build() {
+  true
+}
+
+install() {
+  sudo install -Dm644 minecraft-pi-reborn-client-${version}-amd64.AppImage "${STOWDIR}/${name}/usr/bin/minecraft-pi-reborn-client"
+  sudo chmod +x "${STOWDIR}/${name}/usr/bin/minecraft-pi-reborn-client"
+  sudo wget -q "https://gitea.thebrokenrail.com/TheBrokenRail/minecraft-pi-reborn/raw/branch/master/launcher/data/client/share/applications/com.thebrokenrail.MCPIRebornClient.desktop" -O "${STOWDIR}/${name}/usr/share/applications/com.thebrokenrail.MCPIRebornClient.desktop"
+  sudo wget -q "https://gitea.thebrokenrail.com/TheBrokenRail/minecraft-pi-reborn/raw/branch/master/launcher/data/client/share/metainfo/com.thebrokenrail.MCPIRebornClient.appdata.xml" -O "${STOWDIR}/${name}/usr/share/metainfo/com.thebrokenrail.MCPIRebornClient.appdata.xml"
+  sudo wget -q "https://gitea.thebrokenrail.com/TheBrokenRail/minecraft-pi-reborn/src/branch/master/launcher/data/com.thebrokenrail.MCPIReborn.png" -O "${STOWDIR}/${name}/usr/share/icons/hicolor/256x256/apps/com.thebrokenrail.MCPIRebornClient.png"
```
Comment (Henryws) [inline packages/minecraft-pi-reborn-app/minecraft-pi-reborn-app.pacscript:24]: Can you make these specific commits, and update them every time a new release comes out?

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by cat-master21** [comment]
Almost forgot to contact Devs!

**Review by Henryws — APPROVED** [review]

**Pull request merged**